content/release-notes/2023-05-19-console.md

```diff
@@ -0,0 +1,20 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- Control Plane: Neon's Autoscaling feature now supports fractional (1/4 and 1/2) minimum compute sizes. Previously, the minimum compute size for Autoscaling was 1.
+![Autoscaling fractional compute sizes](/docs/relnotes/fractional_compute_sizes.png)
+  
+  Autoscaling is a [Neon Pro plan](/docs/introduction/pro-plan) feature. For configuration instructions, see [Compute size and Autoscaling configuration](../manage/endpoints#compute-size-and-autoscaling-configuration).
+- API: Added a `synthetic_storage_size` property to the [Projects](https://api-docs.neon.tech/reference/getproject) API, which provides the Neon data storage size in bytes. The value represents  the logical data size plus the amount of Write-Ahead Log (WAL) data stored by Neon. This property is  experimental.
```

Comment:
wasn't this part of getproject already?
i think this release added this property to a different APi method, this one: https://api-docs.neon.tech/reference/listprojectsconsumption

Reply:
Thank you. I will remove it since that API is not publicly available.